spec/features/workflow/delete_draft_after_publishing_spec.rb

```diff
@@ -50,7 +54,7 @@ def then_i_see_the_updated_draft
   end
 
   def then_i_see_the_draft_is_gone
-    expect(page).to have_current_path(documents_path)
+    expect(page).to have_current_path(documents_path(organisation: User.first.organisation_content_id))
```

Comment:
Would it be cleaner to use a regex match here, so that we're not too coupled to our filtering behaviour? I guess there's a slight compromise, as the reason this is here is to try and verify we're on the right page to see something is gone, but then we do have the filtering feature test for that.